.github/workflows/e2e-k3s.yaml

```diff
@@ -9,83 +9,39 @@ on:
 
 jobs:
 
-  image-build-and-push-query-service:
+  e2e-k3s:
     runs-on: ubuntu-latest
     if: ${{ github.event.label.name == 'ok-to-test' }}
```

Comment:
@ybettan  can you include `DOCKER_TAG` env for the job instead of adding it in multiple steps?

Reply:
Good point, updating it.